targets/XPlatCppSdk/source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -0,0 +1,743 @@
+#include <stdafx.h>
+
+#include <jni.h>
+#include <android/log.h>
```

Comment:
>log.h> [](start = 18, length = 6)

Do you need this ?

Reply:
This is for using __android_log_print() to output log. and the __android_log_print are used only Shared Object loading and unloading.
This will be good to keep this log to check the shared object is successfully loaded or not.

---
In reply to: [274693173](https://github.com/PlayFab/SDKGenerator/pull/470#discussion_r274693173) [](ancestors = 274693173)